test/minitest/test_minitest_test.rb

```diff
@@ -848,19 +848,25 @@ def non_verbose
     $VERBOSE = orig_verbose
   end
 
+  def sample_test_case(rand = nil)
+    srand(rand) if rand
```

Comment:
no parens on the srand above please

why is `rand` optional if you use it in 100% of the calls?

Reply:
Don't recall.  I might have called it without a parameter while writing/testing it.  I'll remove both.